_episodes/23-object-oriented.md

```diff
@@ -155,27 +155,30 @@ With a class, if an object is an **instance** of that class (i.e. it was made us
 
 Different programming languages make slightly different guarantees about how strictly the structure will match, but in object oriented programming this is one of the core ideas.
 
-For an example of a class, we're going to step away from the inflammation data for a while to another model.
-Let's create a model of academics publishing papers and we'll add extra functionality to this model as we learn more about Object Oriented Programming in Python.
-Note that this isn't part of our inflammation data example, so make sure you keep this code separate - for example in a different directory, or just make sure you don't commit it to your inflammation repository.
+Let's start with a minimal example of a class representing our patients.
 
-~~~
-class Academic:
+~~~ python
```

Comment:
Feels like there should be an instruction to put this in a file (perhaps with a particular name? Or is this being run within the interpreter for now?

Reply:
Ah, yeah I missed some of the file instructions, will add them